tests/functional/m/modified_iterating.py

```diff
@@ -93,3 +93,15 @@ def update_existing_key():
     for key in my_dict:
         new_key = key.lower()
         my_dict[new_key] = 1  # [modified-iterating-dict]
+
+
+class MyClass:  # pylint: disable=too-few-public-methods
```

Comment:
This is the only class in the test file. you can add the disable flag at the top of the file

Reply:
This is personal preference I think. I like disabling at the occurrence as it allows more freedom in checking for some message further down the file.

But I'll change it here 😄